read_unix.go

```diff
@@ -9,6 +9,9 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// LINE_SEP is the separator for windows or unix systems
+const LINE_SEP = "\n"
```

Comment:
done! no problem - I had only kept it that way given the other const declared below used the allcaps/snakecase. Updated to be idiomatic